### PR TITLE
fix(typings): seeTextEquals, detox, event, recorder and output

### PR DESCRIPTION
**Pull request by Vorobeyko**
## Motivation/Description of the PR

Fix: https://github.com/Codeception/CodeceptJS/issues/2079 , https://github.com/Codeception/CodeceptJS/issues/2024

Applicable helpers:

- [ ] Webdriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe

- Description of this PR, which problem it solves
- A link to the corresponding issue (if applicable).

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [x] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [x] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [x] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: runio.js [hunk 1]
```diff
@@ -86,11 +86,19 @@ Our community prepared some valuable recipes for setting up CI systems with Code
     // generate documentation for helpers outside of main repo
     console.log('Building @codecepjs/detox helper docs');
     const helper = 'Detox';
+    replaceInFile(`node_modules/@codeceptjs/detox-helper/${helper}.js`, (cfg) => {
+      cfg.replace(/CodeceptJS.LocatorOrString/g, 'string | object');
```
Comment (elukoyanov) [inline runio.js]: Why is this feint here?

Whe replace on 'string | object', and then back again?

What will be with not `LocatorOrString`, but  `'string | object'` types?

**Comment by elukoyanov** [comment]
What difference with #2027

File: typings/types.d.ts [hunk 2]
```diff
@@ -9069,26 +9084,33 @@ declare namespace CodeceptJS {
          * Start recording promises
          *
          * @api
+         * @inner
```
[the remark this answers is not in the document]
Comment (Vorobeyko) [reply typings/types.d.ts]: Because ts raise error if not add @inner.

File: lib/index.js [hunk 3]
```diff
@@ -3,70 +3,37 @@
...
  * @alias index
- * @interface
+ * @namespace
  */
 module.exports = {
-  /**
-   * @type {typeof CodeceptJS.Codecept}
-   * @inner
-   */
+  /** @type {typeof CodeceptJS.Codecept} */
```
[the remark this answers is not in the document]
Comment (Vorobeyko) [reply lib/index.js]: =) 
I dont know) Maybe it more beautiful)

**Comment by DavertMik** [comment]
@elukoyanov I didn't find any objections from your side so I am taking the decision to merge it.

**Comment by elukoyanov** [comment]
So, maybe close  Lisa's request?

**Pull request merged**